formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/FormstackSarService.scala

```diff
@@ -77,4 +86,58 @@ object FormstackSarService extends FormstackSar with LazyLogging {
     else
       decode[FormstackFormSubmissionsResponse](response.body)
   }
+
+  private def submissionsById(
+    submissionIdEmails: List[SubmissionIdEmail],
+    accountToken: FormstackAccountToken,
+    encryptionPassword: String): Either[Throwable, List[FormstackSubmissionResponse]] = {
+    submissionIdEmails.traverse { submissionIdEmail =>
+      val response =
+        Http(s"https://www.formstack.com/api/v2/submission/${submissionIdEmail.submissionId}.json")
+          .header("Authorization", accountToken.secret)
+          .param("encryption_password", encryptionPassword)
+          .asString
+
+      if(!response.is2xx) {
+        logger.error(response.body)
+      }
+
+      decode[FormstackSubmissionResponse](response.body)
+    }
+  }
+
+  private def retrieveSubmissionLabels(
```

Comment:
same as above. may be getSubmissionQuestionAnswers?

Reply:
done!